### PR TITLE
Desktop version with @media

### PR DESCRIPTION
**Pull request by SergSkol**
- Add @media
- Adjust navigation, images, margins, etc

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -101,9 +108,61 @@ <h1 class="my-recent-works">My Recent Works</h1>
...
+          <ul class="work-card-tags nobull">
+            <li class="work-card-tag">Ruby on rails</li>
+            <li class="work-card-tag">css</li>
+            <li class="work-card-tag">JavaScript</li>
+            <li class="work-card-tag">html</li>
+          </ul>
+          <button class="work-card-button">See Project</button>
+        </div>
+      </div>
     </section>
```
Comment (Nemwel-Boniface) [inline index.html:162]: - [ ] Kindly ensure that you stick as much as possible to the provided Figma design. Currently, the gap between the individual cards is big which makes the margins/ padding left and right of the cards not to match  the provided Figma design. Kindly look into it.

**This is how I am seeing this section:**

image

**This is the desired Figma design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @SergSkol    :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your portfolio desktop version so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -146,7 +205,7 @@ <h1 class="about-me-title">About me</h1>
         </ul>
       </div>
     </section>
-    <section class="contact-me">
+    <section class="contact-me" id="contact">
```
[the remark this answers is not in the document]
Comment (SergSkol) [reply index.html:208]: Thank you! It's done!

File: index.html (hunk 1 above)
Comment (SergSkol) [reply index.html:162]: Thank you! I've fixed gaps between grid items using width as 87%

File: index.html [hunk 3]
```diff
@@ -169,20 +228,20 @@ <h1 class="about-me-title">About me</h1>
...
+      <li>
+        <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+      </li>
+    </ul>
```
Comment (youmari) [inline index.html:242]: - [ ] nice adding asked changes but you forgot to add a gray line above your social media icons as in the Figma design so please try to add in order to match The Figma design 
- Yours
Screen Shot 2022-10-05 at 15 38 20
-Figma
Screen Shot 2022-10-05 at 15 38 49

File: index.html [hunk 4]
```diff
@@ -169,20 +228,20 @@ <h1 class="about-me-title">About me</h1>
...
+    <section class="email">
+      <p>sps676@gmail.com</p>
```
Comment (youmari) [inline index.html:244]: - [ ] [OPTIONAL]  I see that there are some missing illustration design please make sure to add them to make your portfolio beautiful and also match the Figma design 
- Yours
Screen Shot 2022-10-05 at 15 41 12
- Figma
Screen Shot 2022-10-05 at 15 41 26

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @SergSkol , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (SergSkol) [reply index.html:244]: Thank you! Done!

**Review by christianbiring1 — APPROVED** [review]
Hi @SergSkol , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
APPROVED

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] All requested changes were implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**